doc/maintaining-V8.md

```diff
@@ -0,0 +1,275 @@
+
+Such fixes are tagged with the following labels in the V8 issue tracker:
+
+*   ~~NodeJS-~~Backport-Review ([V8](https://bugs.chromium.org/p/v8/issues/list?can=1&q=label%3ABackport-Review), [Chromium](https://bugs.chromium.org/p/chromium/issues/list?can=1&q=label%3ABackport-Review)): to be reviewed if this is applicable to abandoned branches in use by Node.js. This list if regularly reviewed by the node team at Google to determine applicability to Node.js.
+*   ~~NodeJS-~~Backport-Approved ([V8](https://bugs.chromium.org/p/v8/issues/list?can=1&q=label%3ABackport-Review), [Chromium](https://bugs.chromium.org/p/chromium/issues/list?can=1&q=label%3ABackport-Review)): marks bugs that are deemed relevant to Node.js and should be backported.
+*   ~~NodeJS-~~Backport-Done ([V8](https://bugs.chromium.org/p/v8/issues/list?can=1&q=label%3ABackport-Review), [Chromium](https://bugs.chromium.org/p/chromium/issues/list?can=1&q=label%3ABackport-Review)): Backport for Node.js has been performed already.
+*   ~~NodeJS-~~Backport-Rejected ([V8](https://bugs.chromium.org/p/v8/issues/list?can=1&q=label%3ABackport-Review), [Chromium](https://bugs.chromium.org/p/chromium/issues/list?can=1&q=label%3ABackport-Review)): Backport for Node.js is not desired.
+
+The backlog of issues with such is regularly reviewed by the node-team at Google to shepherd through the backport process. External contributors are welcome to collaborate on the backport process as well. Note that some of the bugs may be security issues and will not be visible to external collaborators.
+
+# Updating V8
+
+Node keeps a vendored copy of V8 inside of deps/ directory. In addition Node may need to float patches that do not exist upstream. This means that some care may need to be taken to update the vendored copy of V8.
+
+## Minor updates (patch level)
+
+Because there may floating patches on the version of V8 in Node.js, it is safest to apply the patch level updates as a patch. For example, imagine that upstream V8 is at 5.0.71.47 and Node.js is at 5.0.71.32. It would be best to compute the diff between these tags on the V8 repository, and then apply that patch on the copy of V8 in Node.js. This should preserve the patches/backports that Node.js may be floating (or else cause a merge conflict).
+
+The rough outline of the process is:
+
+```shell
+# Assuming your fork of Node.js is checked out in $NODE_DIR
+# and you want to update the Node.js master branch.
+# Find the current (OLD) version in
+# $NODE_DIR/deps/v8/include/v8-version.h
+cd $NODE_DIR
+git checkout master
+git merge --ff-only upstream/master
+git checkout -b V8_NEW_VERSION
+curl -L https://github.com/v8/v8/compare/${V8_OLD_VERSION}...${V8_NEW_VERSION}.patch > /tmp/patch-1
+cd deps/v8
+patch -p1 < /tmp/patch-1
+cd ../..
+git commit -am 'deps: update V8 from V8_OLD_VERSION to V8_NEW_VERSION'
```

Comment:
The patch can add new files (often tests).
I use `git add deps/v8 && git commit -m '...'` to not miss anything